components/Article/TrialPayNoteMini.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react'
+import { css } from 'glamor'
+
+import TrialForm from '../Trial/Form'
+import { TRIAL_CAMPAIGN } from '../../lib/constants'
+import withT from '../../lib/withT'
+import {
+  fontStyles,
+  useColorContext,
+  Center,
+  mediaQueries,
+  RawHtml
+} from '@project-r/styleguide'
+
+import BrowserOnly from './BrowserOnly'
+
+const accessCampaignId = TRIAL_CAMPAIGN
+
+const styles = {
+  container: css({
+    padding: 12
+  }),
+  title: css({
+    margin: 0,
+    ...fontStyles.serifTitle32
+  }),
+  lead: css({
+    marginBottom: 0,
+    ...fontStyles.serifRegular18,
+    [mediaQueries.mUp]: {
+      ...fontStyles.serifRegular21
+    }
+  })
```

Comment:
pay note always use interaction font faces, ideally we could just use interaction typography elements from the styleguide